clients/fossid-webapp/src/main/kotlin/FossIdRestService.kt

```diff
@@ -58,6 +62,9 @@ import retrofit2.http.POST
 
 interface FossIdRestService {
     companion object {
+        @JvmStatic
+        val logger = loggerOf(FossIdRestService::class.java)
```

Comment:
Nit: Would have been nice to also use just `log` here as in ORT to ease code search for logging statements. But provided that it's unsure how long we'll keep it, it's also ok for me to stick to this.

Reply:
I thought about exactly the opposite :)
I would like the people to see that this code is special and does not use standard ORT logger, hence the different name.

The day we migrate from log4j, we will start by looking at all dependencies in `build.gradle.kts` files anyway...